pom.xml

```diff
@@ -58,56 +58,43 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 		<repository>
 			<id>spring-maven-release</id>
 			<name>Spring Maven Release Repository</name>
-			<url>http://maven.springframework.org/release</url>
+			<url>https://maven.springframework.org/release</url>
 		</repository>
 		<repository>
 			<id>spring-maven-milestone</id>
 			<name>Spring Maven Milestone Repository</name>
-			<url>http://maven.springframework.org/milestone</url>
+			<url>https://maven.springframework.org/milestone</url>
 		</repository>
 		<repository>
 			<id>spring-roo-repository</id>
 			<name>Spring Roo Repository</name>
-			<url>http://spring-roo-repository.springsource.org/release</url>
-		</repository>
-		<repository>
-			<id>jcenter-release</id>
-			<name>jcenter</name>
-			<url>http://oss.jfrog.org/artifactory/oss-release-local/</url>
-		</repository>
-		<repository>
-			<id>OSGEO GeoTools repo</id>
-			<url>http://download.osgeo.org/webdav/geotools</url>
-		</repository>
-		<repository>
-			<id>codehaus</id>
-			<url>https://repository.codehaus.org/org/codehaus</url>
+			<url>https://spring-roo-repository.springsource.org/release</url>
 		</repository>
 	</repositories>
 	<pluginRepositories>
 		<pluginRepository>
 			<id>spring-maven-release</id>
 			<name>Spring Maven Release Repository</name>
-			<url>http://maven.springframework.org/release</url>
+			<url>https://maven.springframework.org/release</url>
 		</pluginRepository>
 		<pluginRepository>
 			<id>spring-maven-milestone</id>
 			<name>Spring Maven Milestone Repository</name>
-			<url>http://maven.springframework.org/milestone</url>
+			<url>https://maven.springframework.org/milestone</url>
 		</pluginRepository>
 		<pluginRepository>
 			<id>spring-roo-repository</id>
 			<name>Spring Roo Repository</name>
-			<url>http://spring-roo-repository.springsource.org/release</url>
+			<url>https://spring-roo-repository.springsource.org/release</url>
 		</pluginRepository>
 	</pluginRepositories>
 	<!-- Maven Build profile controls whether to use JChem or Indigo -->
 	<profiles>
 		<profile>
-			<id>default</id>
+			<id>jchem</id>
```

Comment:
This is backwards compatable with -P default because it IS the default and so even if you provide a bad profile name like `-P default` you just get a warning and then it uses this anyway.